### PR TITLE
dwm: new package at 6.2

### PR DESCRIPTION
**Pull request by fanboimsft**
dynamic window manager from suckless.org

- [6.2] Latest upstream version.

### CODE REVIEW

File: community/cifs-utils/build [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
```
Comment (dylanaraps) [inline community/cifs-utils/build]:
```sh
./configure \
    --prefix=/usr
```

File: community/dwm/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+#amend install path to /usr
```
Comment (dylanaraps) [inline community/dwm/build]:
```sh
# Amend install path to '/usr'.
sed -i'' "s#/usr/local#/usr#" config.mk
```

File: community/ntfs-3g/build [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
```
Comment (dylanaraps) [inline community/ntfs-3g/build]:
```sh
./configure \
    --prefix=/usr
```

File: community/ntfs-3g/build [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+make
+make DESTDIR="$1" install
+
+#remove docs
```
Comment (dylanaraps) [inline community/ntfs-3g/build]:
```
# Remove docs.
rm -rf "$1/usr/share/doc"
```

**Comment by dylanaraps** [comment]
You've also added `ntfs-3g` to this pull request. Is this a requirement for `dwm`?

**Comment by fanboimsft** [comment]
No both cifs-utils and ntfs-3g are not required for dwm. 

cifs-utils is required to mount samba shares
ntfs-3g is required to read/write ntfs volumes as an user

**Comment by dylanaraps** [comment]
You'll need to split each of these packages into separate pull requests. `xf86-video-amdgpu` should probably go in the core repositories.

**Pull request closed without merging**